FreeRTOS/coreJSON
# Remove changes to achieve 100% coverage

Description
-----------
This PR removes changes in source code , introduced in #146  to achieve 100% branch coverage.

Test Steps
-----------
```
cmake -S .. -B build
make coverage
```
Before applying the patch

```
                       |Lines       |Functions  |Branches
Filename        |Rate     Num|Rate    Num|Rate     Num
=========================================================
core_json.c     | 100%    554| 100%    35| 99.50%    601
=========================================================
     Total:     | 100%    554| 100%    35| 100%      604

```
After applying the patch

```
                       |Lines       |Functions  |Branches
Filename        |Rate     Num|Rate    Num|Rate     Num
======================================================
core_json.c     | 100%    588| 100%    35| 100%    596
======================================================
     Total:     | 100%    588| 100%    35| 100%    596

```

Checklist:
----------



Related Issue
-----------


By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
